src/component/Landing/Main.js

```
@@ -39,100 +52,120 @@ const Main = () => {
-                            "Decorations for Roka, Engagement, Mehendi, Haldi, Sangeet"}
-                        </p>
-
-                        {item?.ctaLabel && item?.ctaHref && (
-                          <a className="btn btn-accent" href={item.ctaHref}>
-                            {item.ctaLabel}
-                          </a>
-                        )}
+            {getTopBannerList?.data?.length > 0
+              ? getTopBannerList?.data?.map((item, i) => {
+                  return (
+                    <div className={`carousel-item ${i == 0 ? "active" : ""}`}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)